<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doodle</class>
 <widget class="QMainWindow" name="Doodle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Doodle</string>
  </property>
  <property name="toolTipDuration">
   <number>11</number>
  </property>
  <widget class="DrawingWidget" name="drawingWidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTipDuration">
    <number>11</number>
   </property>
   <property name="statusTip">
    <string>10</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="newMenu"/>
    <addaction name="openMenu"/>
    <addaction name="saveMenu"/>
    <addaction name="othersaveMenu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>편집</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="newMenu">
   <property name="text">
    <string>새로 만들기</string>
   </property>
  </action>
  <action name="openMenu">
   <property name="text">
    <string>열기</string>
   </property>
  </action>
  <action name="saveMenu">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="othersaveMenu">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingWidget</class>
   <extends>QWidget</extends>
   <header>Ui\DrawingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="doodle.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>changed()</signal>
 </slots>
</ui>
